HandEyeCalibration/UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp

```diff
@@ -0,0 +1,95 @@
+/*
+Utilize the result of eye-in-hand calibration to transform (picking) point
+coordinates from the camera frame to the robot base frame.
+*/
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+
+#include <opencv2/core/core.hpp>
+#include <cmath>
+#include <iostream>
+#include <filesystem>
+
+Eigen::MatrixXd cv2eigen(const cv::Mat &cv_mat)
+{
+    if(cv_mat.dims > 2)
+    {
+        throw "Invalid matrix dimensions. Expected 2D.";
+    }
+
+    Eigen::MatrixXd eigen_mat(cv_mat.rows, cv_mat.cols);
+
+    for(int i = 0; i < cv_mat.rows; i++)
+    {
+        for(int j = 0; j < cv_mat.cols; j++)
+        {
+            eigen_mat(i, j) = cv_mat.at<double>(i, j);
+        }
+    }
+
+    return eigen_mat;
+}
+
+
+cv::Mat read_transform(const std::string file_name)
```

Comment:
Functions should also be camelCase. The exception is cv2eigen. I don't know what is a good way to make that one camelCase. @knatten any idea?

Reply:
Maybe call it `cvToEigen`? `To` might be better than `2` anyway?